app/bundles/DashboardBundle/Views/Dashboard/index.html.php

```diff
@@ -61,6 +61,14 @@
     'customButtons' => $buttons,
 ]));
 ?>
+<?php if (true === $phpVersion['isOutdated']): ?>
+<div class="row pt-md pl-md">
```

Comment:
If we remove row class attribute we get same padding, as dashboard widgets bellow

![image](https://user-images.githubusercontent.com/462477/104820320-50148d80-5834-11eb-9693-1b0898fc1bef.png)

Reply:
Done, thanks for the suggestion @kuzmany 🚀